src/lib/Components/Inbox/Conversations/ConversationSnippet.tsx

```diff
@@ -169,7 +169,7 @@ export class ConversationSnippet extends React.Component<Props, any> {
                   <MetadataText>
                     {date}
                   </MetadataText>
-                  <UnreadIndicator />
+                  {conversation.is_last_message_to_user && !conversation.last_message_open && <UnreadIndicator />}
```

Comment:
B/c the unread indicator is supposed to indicate if there's new content for the user. So if the last message in a thread is the user sending something to the gallery, we wouldn't want to show an unread indicator next to that thread.